tpl/tplimpl/embedded/templates/shortcodes/youtube.html

```diff
@@ -4,7 +4,17 @@
 {{- $id := .Get "id" | default (.Get 0) -}}
 {{- $class := .Get "class" | default (.Get 1) -}}
 {{- $title := .Get "title" | default "YouTube Video" }}
+{{- $start := .Get "start" | default "" -}}
+{{- $autoplay := .Get "autoplay" | default "" -}}
+{{- $queryComponents := slice -}}
+{{- if $start -}}
+  {{- $queryComponents = $queryComponents | append "start" $start -}}
+{{- end -}}
+{{- if $autoplay -}}
+  {{- $queryComponents = $queryComponents | append "autoplay" 1 -}}
+{{- end -}}
```

Comment:
```suggestion
{{- $queryComponents := slice -}}
{{- with .Get "start" -}}
  {{- $queryComponents = $queryComponents | append "start" . -}}
{{- end -}}
{{- if .Get "autoplay" -}}
  {{- $queryComponents = $queryComponents | append "autoplay" 1 -}}
{{- end -}}
```

Reply:
@septs both of these changes looks good, but it makes it extremely hard to review a specific change with unrelated changes coming in from the side line.